lapis/nginx.lua

```diff
@@ -109,7 +109,8 @@ local ngx_req = {
   end,
   parsed_url = function(t)
     local uri = ngx.var.request_uri
-    uri = uri:match("(.-)%?") or uri
+    local pos = uri:find("?")
```

Comment:
also those two lines could be turned into something like `local uri = uri:sub(1, (uri:find("?", 1, true) or 0) -1)